content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -14,6 +14,18 @@ The `kube-proxy` component is responsible for implementing a _virtual IP_
 mechanism for {{< glossary_tooltip term_id="service" text="Services">}}
 of `type` other than
 [`ExternalName`](/docs/concepts/services-networking/service/#externalname).
+Kube-proxy watches the Kubernetes {{< glossary_tooltip
+term_id="control-plane" text="control plane" >}} for the addition and
+removal of Service and EndpointSlice {{< glossary_tooltip
+term_id="object" text="objects." >}} For each Service, kube-proxy
```

Comment:
```suggestion
term_id="object" text="objects" >}}. For each Service, kube-proxy
```

Reply:
oh, good, I had just moved this from somewhere else and was like "wow... that's... a style choice"